### ISSUE TITLE
Minor hickup in boot process

### ISSUE THREAD

**Issue by thorian93** [sinner-/kickstart-fedora-workstation#1]
Hi Sina,
first off a big shout out thanks for your work, you saved me some headache and effort with your work!

I only have one minor note for you.
Maybe that was obvious for you but in the boot process I missed something that took me 30 minutes to figure out: In the boot options line there was a directive called `inst.stage2=`. I had to put the `inst.ks=` directive in front of that so the kickstart file would be loaded from the webserver. Maybe you want to include that note in your README? Or was I missing something obvious? If so I would appreciate a brief explanation.

Thanks and have a goods one!

**Comment by sinner-** [comment]
Hi @thorian93,

That's weird, I always put `inst.ks=` at the end of the boot line, without any issue. I don't remember ever even seeing `inst.stage2=`.

Are you using the netinst ISO? Or the normal installer ISO? I always use the netinst ISO, if that makes any difference.

**Comment by thorian93** [comment]
Good morning, yes I agree, weird is the right word.

I initially used the normal F29 live ISO and when that did not work I downloaded the F29 netinstaller from the official homepage and tried that, but the kernel params were the same. I actually have no clue what that `stage2` directive does, as I just started digging into kickstart actually. I will omit that directive later completely and see what that does.

I will keep you posted.

**Comment by thorian93** [comment]
Hi Sina, so here is my update:
The default parameters I see when booting the netinstall are: `inst.stage2=hd:LABEL=Fedora-WS-dvd-x86_64-29 quiet`
The beginning of the line depends on EFI or BIOS boot. It does not matter which one I use apparently.
What I observed is the following:
Inserting the `inst.ks` directive after `quiet` results in the directive being ignored.
Inserting the `inst.ks` directive before `quiet` works.
Replacing the `inst.stage2` directive with `inst.ks` works.
So to sum it up: If you could just add a note about this scenario in your README then we are good I think. :)

**Comment by thorian93** [comment]
One more update: I tested all this in a VM. When I went to a hardware machine, I had to leave the `inst.stage2` directive and add the `inst.ks`directive in front of it, to make it work.

**Comment by sinner-** [comment]
Updated the README, thankyou.

Happy New Year.
